index.html

```diff
@@ -1,29 +1,163 @@
+                   <li><button class="languages-List" type="submit">css</button></li>
+                    <li><button class="languages-List" type="submit">javascript</button></li>
+                </ul>
+                <button class="btn-Project" type="submit">See Project</button>
+
+            </section> <section class="works">
+                <div>
+                    <img class="snapshoot" src="images/Snapshoot-Portfolio3.svg" alt="Snapshoot-Image">
+                </div>
+                <h2 class="tonic">Tonic</h2>
+                <ul class="sub-Points">
+                    <li class="canopy">CANOPY</li>
+                    <li class="sub-Text">Back End Dev</li>
+                    <li class="sub-Text">2015</li>
+                </ul>
+                <p class="description-Text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li><button class="languages-List" type="submit">html</button></li>
+                   <li><button class="languages-List" type="submit">css</button></li>
+                    <li><button class="languages-List" type="submit">javascript</button></li>
+                </ul>
+                <button class="btn-Project" type="submit">See Project</button>
+
+            </section> 
+            <section class="works">
+                <div>
+                    <img class="snapshoot" src="images/Snapshoot-Portfolio4.svg" alt="Snapshoot-Image">
+                </div>
+                <h2 class="tonic">Multi-Post Stories</h2>
+                <ul class="sub-Points">
+                    <li class="canopy">CANOPY</li>
+                    <li class="sub-Text">Back End Dev</li>
+                    <li class="sub-Text">2015</li>
+                </ul>
+                <p class="description-Text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li><button class="languages-List" type="submit">html</button></li>
+                   <li><button class="languages-List" type="submit">css</button></li>
+                    <li><button class="languages-List" type="submit">javascript</button></li>
+                </ul>
+                <button class="btn-Project" type="submit">See Project</button>
+            </section>
+            <section class="about-Myself">
+                <h2 class="about-Text">About<br>Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <h2 class="connect">LET'S CONNECT</h2>
+                <ul>
+                    <li><img class="section1-Icons" src="images/Frame1.svg" alt="social-Media-Icons"></li>
```

Comment:
[REQUIRED] A great trick you did here with inserting `social` icons with the `screenshot` :smile: However, please do not do this :pray:. After all, they are supposed to be actual `links` to your social media profiles. Please kindly  a `<ul/>` tag to group all your social icons, and use  `<li/>` per icon; each icon should have a link (`<a/>` tag)